src/assert.js

```diff
@@ -414,7 +430,8 @@ class Assert {
 					result,
 					actual,
 					expected,
-					message
+					message,
+					assertionType: assertionType
```

Comment:
Nitpick: just `assertionType` instead of `assertionType: assertionType` to be consistent